File: 2018/J1.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int input1 = scanner.nextInt();
        int input2 = scanner.nextInt();
        int input3 = scanner.nextInt();
        int input4 = scanner.nextInt();
        boolean telemarketer = false;


        if (input1 == 8 || input1 == 9 && input4 == 8 || input4 == 9) {
            if (input2 == input3) {
                telemarketer = true;
            }
        }
        if (telemarketer) {
            System.out.println("ignore");
        }
        if (!telemarketer) {
            System.out.println("answer");
        }
    }
}

File: 2018/J1.py
# Alec Dewulf
# 2018 J1 "Telemarketer or not?"
# February 10, 2020

n1 = int(input())
n2 = int(input())
n3 = int(input())
n4 = int(input())

if n1 == 8 or n1 == 9:
    if n4 == 8 or n4 == 9:
        if n3 == n2:
            print('ignore')
        else:
            print('answer')
    else:
        print('answer')
else:
    print('answer')
